src/Nancy/project.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "2.0.0-*",
+
+    "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
+    "authors": [ "Andreas H�kansson, Steven Robbins and contributors" ],
+    "tags": [ "Nancy" ],
+    "projectUrl": "http://nancyfx.org",
+    "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
+    "iconUrl": "http://nancyfx.org/nancy-nuget.png",
+
+    "frameworks": {
+      "dnx451": {
+        "frameworkAssemblies": {
+          "System.Xml" : "4.0.0.0"
+        }
+      },
+        "dnxcore50": {
+          "dependencies": {
+            "Microsoft.CSharp": "4.0.0",
+            "System.Collections.Concurrent": "4.0.10",
+            "System.Collections.Specialized": "4.0.0",
+            "System.ComponentModel.TypeConverter": "4.0.0",
+            "System.Console": "4.0.0-beta-23516",
+            "System.Data.Common": "4.0.0",
+            "System.Diagnostics.Tools": "4.0.0",
+            "System.Dynamic.Runtime": "4.0.0-beta-23516",
+            "System.IO": "4.0.10-beta-23516",
+            "System.IO.UnmanagedMemoryStream": "4.0.0",

```

Comment:
looks like we aren't, it was in jchannon's dnx branch but i can't tell why, I will revisit this and make sure all the packages are needed.